data/assets/scene/solarsystem/missions/insight/edl.asset

```diff
@@ -1,6 +1,8 @@
 asset.require("spice/base")
 local sun = asset.require("scene/solarsystem/sun/sun")
 local mars = asset.require("scene/solarsystem/planets/mars/mars")
+asset.require("./actions")
```

Comment:
Would it make sense to switch these two lines?  Would fit our include-file paradigm in cpp files better to go form general -> specific